src/modules/client/pages/ni/pay/ContractEnds.vue

```diff
@@ -188,6 +190,12 @@ export default {
         NotifyNegative('Erreur lors de la création des soldes tout compte.');
       }
     },
+    shouldDisplayDetails (id, details) {
```

Comment:
Tu peux la mettre dans payMixin il me semble

Reply:
Les deux ne sont pas les même, dans une c'est this.draftPay et l'autre this.draftFinalPay

Reply:
Que penses-tu d'utiliser la meme methode en passant en parametre this.draftPay ou this.draftFinalPay ?